testing/TestHarness/TestHarness.Windows/TestHarness.Windows.csproj

```diff
@@ -53,6 +53,7 @@
 		<PackageReference Include="Uno.Toolkit.WinUI.Material" />
 		<PackageReference Include="Uno.Toolkit.WinUI" />
 		<PackageReference Include="Uno.WinUI.MSAL" />
+		<PackageReference Include="Uno.WinUI" />
```

Comment:
I'm curious why this reference is needed here...

Reply:
This was an attempt to try to fix the build error